utilities/option_change_migration/option_change_migration_test.cc

```diff
@@ -49,6 +49,7 @@ class DBOptionChangeMigrationTests
 #ifndef ROCKSDB_LITE
 TEST_P(DBOptionChangeMigrationTests, Migrate1) {
   Options old_options = CurrentOptions();
+  old_options.max_open_files = -1;
```

Comment:
Does it work to only make this change when `compaction_style == CompactionStyle::kCompactionStyleFIFO`? We'd probably also need to conditionally modify `new_options` below if we conditionally modified `old_options` here.

Reply:
@ajkr yep! will do. i planned to ping you on this one as i wasn't super confident that this particular change was in the right place heh